### PR TITLE
Inline several functions from util.c

### PR DESCRIPTION
**Pull request by khwilliamson**
These trivial functions will likely get optimized out.  But I'm unsure if inlining all of them is a good idea

### CODE REVIEW

File: inline.h [hunk 1]
```diff
@@ -3698,6 +3740,46 @@ Perl_my_strlcpy(char *dst, const char *src, Size_t size)
...
+*/
+
+PERL_STATIC_INLINE bool
+Perl_sv_destroyable(pTHX_ SV *sv)
+{
+    PERL_UNUSED_CONTEXT;
+    PERL_UNUSED_ARG(sv);
+    return TRUE;
+}
+
```
Comment (tonycoz) [inline inline.h]: These are only called via function pointers, there's no point to inlining them

Comment (khwilliamson) [reply inline.h]: PR updated to not include these two.

File: embed.fnc [hunk 2]
```diff
@@ -627,7 +627,7 @@ CRTopd	|Malloc_t|realloc	|Malloc_t where 			\
 				|MEM_SIZE nbytes
 CTopd	|Free_t |mfree		|Malloc_t where
 
-CTpdR	|void * |get_context
+CTsdR	|void * |get_context
```
Comment (tonycoz) [inline embed.fnc]: Shouldn't these have the `i` flag?

File: inline.h [hunk 3]
```diff
@@ -3624,6 +3624,48 @@ Perl_savesharedsvpv(pTHX_ SV *sv)
     return savesharedpvn(pv, len);
 }
 
+#ifndef PERL_GET_CONTEXT_DEFINED
+
+/*
+=for apidoc_section $embedding
+=for apidoc get_context
+
+Implements L<perlapi/C<PERL_GET_CONTEXT>>, which you should use instead.
```
Comment (tonycoz) [inline inline.h:3659]: > ..\av.c(1208): error C2129: static function 'void *Perl_get_context(void)' declared but not defined

The version from `win32/win32thread.c` should probably be inlined too, possibly in `win32/win32thread.h`

**Comment by jkeenan** [comment]
> These trivial functions will likely get optimized out. But I'm unsure if inlining all of them is a good idea

@khwilliamson, this pull request is failing to compile during `make` on Win32.  Could you investigate?

**Comment by demerphq** [comment]
'f' has to be the most minimal commit message I have seen so far. ;-)

Typo?

File: embed.fnc [hunk 4]
```diff
@@ -529,7 +529,9 @@
 :   's'  Static function, but function in source code has a Perl_ prefix:
 :
 :	 This is used for functions that have always had a Perl_ prefix, but
-:	 have been moved to a header file and declared static.
+:	 have been moved to a header file and declared static.  If those
+:	 functions are also inlined, the 'i' or 'I' flags should also be
```
Comment (demerphq) [inline embed.fnc]: FWIW, this is not correct. the 's', 'S', 'i' and 'I' (capital I) flags are mutually exclusive. You can only have one. I think you have been mislead as there is a bug in embed.pl that uses the first one in the flags definition. I just happened to stumble on this bug myself independently of this ticket, and will push a PR to fix it in the next minutes. Note the code that looks like this: 

```
                $type = {
                    'S' => 'STATIC',
                    's' => 'STATIC',
                    'i' => 'PERL_STATIC_INLINE',
                    'I' => 'PERL_STATIC_FORCE_INLINE'
                }->{$1};
```

**Review by demerphq — CHANGES_REQUESTED** [review]

File: embed.fnc (hunk 4 above)
Comment (demerphq) [reply embed.fnc]: See https://github.com/Perl/perl5/pull/20755

File: embed.fnc (hunk 2 above)
Comment (khwilliamson) [reply embed.fnc]: Yes

File: embed.fnc [hunk 5]
```diff
@@ -6108,11 +6107,13 @@ Cp	|int	|do_spawn	|NN char *cmd
 Cp	|int	|do_spawn_nowait|NN char *cmd
 #endif /* defined(VMS) || defined(WIN32) */
 #if defined(WIN32)
+CTsdR	|void * |get_context
```
Comment (tonycoz) [inline embed.fnc]: I suspect the Win32 failures are because the Win32 Perl_get_context() isn't static.

Comment (demerphq) [reply embed.fnc]: FWIW, I fixed that as part of the conflict resolution on embed.fnc

**Comment by demerphq** [comment]
@khwilliamson I took the liberty of resolving the embef.fnc conflicts and resolved the win32 get_context not being static that was  raised by @tonycoz above at the same time and forced pushed.

**Review by demerphq — COMMENTED** [review]
Looks ok to me now, but lets see how the tests work out.

**Review by demerphq — APPROVED** [review]
Approving to clear the "request changes" status.

**Review by tonycoz — APPROVED** [review]

**Pull request merged**